src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -1025,9 +1038,40 @@ async def _handle_message(self, message: RTVIMessage):
             logger.warning(f"Exception processing message: {e}")
 
     async def _handle_client_ready(self, request_id: str):
+        logger.info("_handle_client_ready")
```

Comment:
This should probably be `debug` and something like `Received client-ready` or something like that.

Reply:
Done!